static/js/redux/ui/advisor_menu.jsx

```diff
@@ -0,0 +1,114 @@
+/*
+Copyright (C) 2017 Semester.ly Technologies, LLC
+Semester.ly is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+Semester.ly is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+*/
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import ClickOutHandler from 'react-onclickout';
+import ReactTooltip from 'react-tooltip';
+
+class AdvisorMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {showDropdown: false};
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+    this.hideDropDown = this.hideDropDown.bind(this);
+    this.toggleDropdown = this.toggleDropdown.bind(this);
+  }
+
+  toggleDropdown() {
+    this.setState({showDropdown: !this.state.showDropdown});
+  }
+
+  hideDropDown() {
+    this.setState({showDropdown: false});
+  }
+
+  render() {
+    const { addRemoveAdvisor } = this.props;
+
+    const toggleAdvisorMenuBtn = (
+        <div style={{margin: "right", marginTop: "5px"}}>
+          <button
+              className="save-timetable add-button"
+              data-for="add-btn-tooltip"
+          >
+            <i className="fa fa-user-plus" />
+          </button>
+        </div>
+    );
+
+    const addRemoveBtn = (advisor, added) => {
+      return (
+          <div className="add-button">
+            <button
+                onClick={() => addRemoveAdvisor(advisor, added)}
+                className="save-timetable add-button"
+                data-tip
+                data-for="add-btn-tooltip"
+            >
+              {/*if (this.props.addedAdvisors)*/}
+              <i className={classNames('fa', { 'fa-plus': !added, 'fa-check': added })} />
+            </button>
+
+            <ReactTooltip
+                id="add-btn-tooltip"
+                class="tooltip"
+                type="dark"
+                place="bottom"
+                effect="solid"
+            >
+              <span>Add or Remove Advisor</span>
+            </ReactTooltip>
+          </div>
+      );
+    }
+
+    const advisorList = (this.props.advisors.length > 0) ?
```

Comment:
When you aren't on any particular semester (aka when selected_semester = null) the list of advisors still shows up above the forum.

Reply:
you can pass this `selected_semester` as a prop and then check that if it is null, we don't display the list. Something like that.

Reply:
this list is also consistent if you switch semesters - but i think in the backend it is semester-based. So, if you add an advisor to one semester, you aren't adding them to all of your transcripts. Not sure.